db/migrate/20231204163029_create_poly_metadata.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+#
+# Add polymorphic metadata table to flexibly hold key value pairs
+# See concern metadatable
```

Comment:
Did there used to be a concern and then you got rid of it?

Reply:
For Comments there is a commentable concern, here:
`app/models/concerns/commentable.rb`
I think I was going to make one of those for metadatable, but then ended up only needing one line on the model, so it seemed excessive. What do you think? I don't get the scope code on commentable, and the 2 other methods are specific to comments.

Reply:
Yeah the scope code is confusing. Looks like it allows you to call e.g. `Plate.with_comments.group` (as well as just `Plate.with_comments` 
(https://stackoverflow.com/questions/39306433/why-is-this-scopes-lambda-followed-by-a-block)

I've had a play in the console and haven't got it to work yet though. Looking through SS & Limber, I can't see `with_comments` used anywhere.

Reply:
In summary, I think I agree, having a concern seems excessive here! We can always add one later if we want to add shared functionality.